src/check_jsonschema/catalog.py

```diff
@@ -88,7 +88,10 @@ def _githubusercontent_url(owner: str, repo: str, ref: str, path: str) -> str:
         "url": "https://json.schemastore.org/github-action",
         "hook_config": {
             "name": "Validate GitHub Actions",
-            "files": ["action.(yml|yaml)", r"\.github/actions/(.+/)?action.(yml|yaml)"],
+            "files": [
+                "action.(yml|yaml)",
```

Comment:
I think you want this one too.
```suggestion
                "action\.(yml|yaml)",
```

Reply:
Hmm this probably slipped my attention because it wasn't a raw string...

In any case, in the commit below I changed `"action.(yml|yaml)"` to `r"action\.(yml|yaml)"`.